plugins/caches.js

```diff
@@ -404,14 +408,18 @@ exports.plugin = {
                     }
 
                     return reject(err);
-                })).then(() => h.response().code(200))
-                    .catch((err) => {
-                        if (err === 'Permission denied') {
-                            return boom.forbidden(err);
-                        }
+                })).then(() => {
+                    request.log([cachePath, 'info'], 'Successfully deleted a cache');
 
-                        return h.response().code(500);
-                    });
+                    return h.response().code(200);
```

Comment:
[imo] Is it better to use status code 204, because success of delete?

Reply:
204 is reasonable. fixed.
https://stackoverflow.com/a/29574521